samples/apps/copilot-chat-app/webapp/src/components/chat/chat-list/NewBotMenu.tsx

```diff
@@ -19,14 +19,13 @@ export const NewBotMenu: FC = () => {
     const fileHandler = useFile();
     // It needs to keep the menu open to keep the FileUploader reference
     // when the file uploader is clicked.
-    const [isNewBotMenuOpen, setIsNewBotMenuOpen] = useState(false);
```

Comment:
We need to keep the menu open to have a handle of the FileUploader reference. Last time I tested, removing this will cause the bot upload feature to fail.

Reply:
Thanks for catching. It causes weird UI bugs because the menu stays persisted. I’ll try figuring out how to fix the document upload while keeping the menu closed

Reply:
@TaoChenOSU, looks like the bug was existed because the FileUploader was disposed whenever the menu component was hidden. I moved the FileUploader component up to the ChatList component and it now works correctly.